frontend/src/views/WasteActionsPage/WasteActionPage.vue

```diff
@@ -102,6 +121,18 @@ export default {
         }
       })
     },
+    hasActionsDone() {
+      return this.wasteAction?.actions?.filter((action) => action.isDone).length > 0
+    },
+    actionsDone() {
```

Comment:
`canteensDoneAction` ? J'essaye de capturer que "this variable contains a list of canteens who have implemented this waste action"

Reply:
yes t'as raison c'est plus clair !